recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import get
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd, cross_building, stdcpp_library
+
+class mysql_connector_cxxRecipe(ConanFile):
```

Comment:
We prefer pascal case for the class as a convention
Also, the [`package_type`](https://docs.conan.io/2/reference/conanfile/attributes.html#package-type) attribute is missing

Reply:
thanks solved